content/types/_paginated_result.textile

```diff
@@ -164,8 +168,17 @@ channel.history { paginatedResult, error in
     print("Page 0 item 0: \((paginatedResult.items[0] as! ARTMessage).data)")
     paginatedResult.next { nextPage, error in
         let nextPage = nextPage!
-        print("Page 0 item 0: \((nextPage.items[1] as! ARTMessage).data)")
+        print("Page 1 item 1: \((nextPage.items[1] as! ARTMessage).data)")
         print("Last page? \(nextPage.isLast())")
     }
 }
 ```
+
+```[go]
+  page0, err := channel.History(nil)
+  fmt.Println("Page. 0 item 0: %s\n", page0.Messages[0].Data)
+  page1, err := page0.Next()
+  fmt.Println("Page. 1 item 1: %s\n", page1.Messages[1].Data)
+  fmt.Println("Last page? %s\n", page1.IsLast())
```

Comment:
Extra white space beneath.  Also not sure that method exists.

Reply:
Regarding the missing methods for HTTPPaginatedResult and such, I've hidden them for now, created an issue in the go lib (https://github.com/ably/ably-go/issues/120), and marked them with `GO_REPLACEMENT`. Once the issue is resolved, that should hopefully be enough reminders to make the go for these visible again.